src/encoded/static/components/cart/cart.js

```diff
@@ -822,6 +824,66 @@ FileFacets.defaultProps = {
 };
 
 
+/**
+ * Display a button that links to experiment search results. This *only* works for Experiments, not
+ * FCCs for now.
+ */
+const MAX_SEARCHABLE_DATASETS = 125; // Maximum number of datasets for search-results links.
```

Comment:
Where did you get 125 from? The ticket does not mention it. I assume you obtained it empirically.

Reply:
I determined it empirically. I mentioned it in the ticket comments.

Reply:
@forresttanaka Thanks. I only looked at the description. I saw it after I looked at the comments.